test/parallel/test-readline-interface.js

```diff
@@ -527,6 +527,154 @@ function isWarned(emitter) {
+      fi.emit('keypress', '.', { ctrl: true, name: 'right' });
+      cursorPos = rli._getCursorPos();
+      assert.strictEqual(cursorPos.rows, 0);
+      assert.strictEqual(cursorPos.cols, 16);
+      fi.emit('keypress', '.', { meta: true, name: 'f' });
+      cursorPos = rli._getCursorPos();
+      assert.strictEqual(cursorPos.rows, 0);
+      assert.strictEqual(cursorPos.cols, 19);
+      rli.close();
+    }
+
+    {
+      // deleteWordLeft
+      [
+        { ctrl: true, name: 'w' },
+        { ctrl: true, name: 'backspace' },
+        { meta: true, name: 'backspace' }
+      ]
+        .forEach((deleteWordLeftKey) => {
+          let fi = new FakeInput();
+          let rli = new readline.Interface({
+            input: fi,
+            output: fi,
+            prompt: '',
+            terminal: terminal
+          });
+          fi.emit('data', 'the quick brown fox');
+          fi.emit('keypress', '.', { ctrl: true, name: 'left' });
+          let called = false;
+          rli.on('line', function(line) {
+            called = true;
+            assert.strictEqual(line, 'the quick fox');
+          });
+          fi.emit('keypress', '.', deleteWordLeftKey);
+          fi.emit('data', '\n');
+          assert.ok(called);
+          rli.close();
+
+          // no effect if pressed at beginning of line
+          fi = new FakeInput();
+          rli = new readline.Interface({
+            input: fi,
+            output: fi,
+            prompt: '',
+            terminal: terminal
+          });
+          fi.emit('data', 'the quick brown fox');
+          fi.emit('keypress', '.', { ctrl: true, name: 'a' });
+          called = false;
+          rli.on('line', function(line) {
+            called = true;
+            assert.strictEqual(line, 'the quick brown fox');
+          });
+          fi.emit('keypress', '.', deleteWordLeftKey);
+          fi.emit('data', '\n');
+          assert.ok(called);
+          rli.close();
+        });
+    }
+
+    {
+      // deleteWordRight
+      [
+        { ctrl: true, name: 'delete' },
+        { meta: true, name: 'delete' },
+        { meta: true, name: 'd' }
+      ]
+        .forEach((deleteWordRightKey) => {
+          let fi = new FakeInput();
+          let rli = new readline.Interface({
+            input: fi,
+            output: fi,
+            prompt: '',
+            terminal: terminal
+          });
+          fi.emit('data', 'the quick brown fox');
+          fi.emit('keypress', '.', { ctrl: true, name: 'left' });
+          fi.emit('keypress', '.', { ctrl: true, name: 'left' });
+          let called = false;
+          rli.on('line', function(line) {
+            called = true;
+            assert.strictEqual(line, 'the quick fox');
+          });
+          fi.emit('keypress', '.', deleteWordRightKey);
+          fi.emit('data', '\n');
+          assert.ok(called);
+          rli.close();
+
+          // no effect if pressed at end of line
```

Comment:
Super tiny nit - would you be so kind and upper case all beginnings of comments?